libraries/chain/smt_evaluator.cpp

```diff
@@ -278,6 +278,17 @@ void smt_setup_emissions_evaluator::do_apply( const smt_setup_emissions_operatio
             "New token emissions must begin no earlier than ${s} seconds after the last emission. Last emission time: ${end}",
             ("s", SMT_EMISSION_MIN_INTERVAL_SECONDS)("end", *end_time) );
       }
+      
+     for ( const auto& e : o.emissions_unit.token_unit )
+     {
+        if (smt::unit_target::is_account_name_type( e.first ))
```

Comment:
Try to keep spacing within conditionals consistent with the rest of the application.